doc/lua_api.txt

```diff
@@ -1476,6 +1476,9 @@ Item metadata only contains a key-value store.
 Some of the values in the key-value store are handled specially:
 
 * `description`: Set the itemstack's description. Defaults to idef.description
+* `color`: A `ColorString`, which sets the stack's color.
+* `paletteindex`: If the item has a palette, this is used to get the
```

Comment:
When colorization for nodes was added, the drawtype was "colored wallmounted" and "colored facedir", but then @paramat pointed out that they should be called "colorwallmounted" and "colorfacedir". This is why I wanted to keep this key name short. There is no naming convention I know of.

Other devs and most importantly modders should decide this name, could you please comment there?